meshtastic/interfaces/ble/compatibility_service.py

```
@@ -0,0 +1,279 @@
+            if callable(is_alive) and not _is_unconfigured_mock_callable(is_alive):
+                try:
+                    alive_result = is_alive()
+                except Exception:  # noqa: BLE001 - enqueue path is best effort
+                    alive_result = False
+                thread_is_alive = (
+                    alive_result if isinstance(alive_result, bool) else False
+                )
+            if thread_is_alive and put_nowait_callback is not None:
+                try:
+                    put_nowait_callback(callback)
+                    return True
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)